server/src/main/java/com/defold/extender/ExtenderYamlSafeConstructor.java

```diff
@@ -0,0 +1,23 @@
+package com.defold.extender;
+
+import org.yaml.snakeyaml.constructor.Construct;
+import org.yaml.snakeyaml.constructor.Constructor;
+import org.yaml.snakeyaml.nodes.Tag;
+
+class ExtenderYamlSafeConstructor extends Constructor {
+
+	public ExtenderYamlSafeConstructor() {
+		super();
+		Construct c = this.yamlConstructors.get(null); // ConstructYamlObject
```

Comment:
I'm not entirely happy with this. The ConstructYamlObject is an inner protected class of the parent and I would prefer to create an instance of it here instead of relying on the fact that it was set in the parent class for the null key....

I'm getting "error: Constructor.ConstructYamlObject() has protected access in Constructor.ConstructYamlObject"